### ISSUE TITLE
GTCEU EU resource type

### ISSUE THREAD

**Issue by bigchunges402960402** [TeamDman/SuperFactoryManager#126]
i have been trying to use factory manager to move eu(gregtech's energy) to a battery can someone tell how to do it? or is there no support for it yet

**Comment by TeamDman** [comment]
There is no EU support yet :(

**Comment by bigchunges402960402** [comment]
how long will it take to add eu support? Also support for gasses and maybe gregtech machines would be hella nice tbh

**Comment by Burkino** [comment]
Gas should already be supported.
Mekanism gas is `gas:*:*`, but every other mod gas is just a fluid.

**Comment by bigchunges402960402** [comment]
yeah i found out on my own but how long will trying to port eu to SFM? Or is it possible to add a addon that adds that?

**Comment by TeamDman** [comment]
🤷
